others/sko/css/style1.css

```diff
@@ -0,0 +1,215 @@
+body {
+    font: 14px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+    background: #080e24 url(http://neography.com/experiment/circles/solarsystem/bg.jpg) repeat;
+    color: #626668;
+}
+
+
+ul.solarsystem {
+    text-indent: -9999px;
+    position: relative;
+    height: 640px;
+    list-style: none;
+    -webkit-transition: all 0.09s ease-in;
+    /*-moz-transition: all 0.09s ease-in;*/
```

Comment:
Depending on the work team culture, commenting out code may not be allowed for production code. Just FYI. But it's okay here